docs/aerodromes/classc/Tindal.md

```diff
@@ -4,7 +4,78 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | Callsign       | Frequency        | Login ID              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| **Tindal ADC**    | **Tindal Tower**  | **119.700**         | **TN_TWR**        |
+| **Tindal SMC**    | **Tindal Ground**  | **135.850**        | **TN_GND**        |
+| **Tindal ACD**    | **Tindal Delivery**  | **128.100**      | **TN_DEL**       |
+| Tindal ATIS    |   |  124.000        | YPTN_ATIS       |
+
+## Airspace
+
+TN ADC owns the Class C airspace **in the TN CTR** within 10nm of the YPTN ARP from `SFC` to `A025`. This airspace is primarily used for military circuits and initial and pitch approaches.
+
+### SID Selection
+Aircraft planned via **DN**, **LAREB**, **GREGA**, **DOSAM**, **MILIV**,**MIGAX**,**DAPMA** shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
+Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure.
+
+#### Gate Departures (All Military Aircraft)
```

Comment:
Maybe a graphic explaining this would be better.

Reply:
For which part, Ill make one for the gates at least.

Reply:
Does that help now?